packages/orbit-components/src/Tabs/components/TabPanel/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
It took me a while to understand what `TabPanel` would be. I honestly think this is overcomplicating the component. To me, the Tab component is just the clickable Tab. The content container is something else, and it can be anything. I don't think we need a component for that.